cmake/HPX_AddConfigTest.cmake

```diff
@@ -74,6 +74,7 @@ macro(add_hpx_config_test variable)
 
       set(_msg "${_msg} - ${_run_msg}")
     else()
+      message("... ${${variable}_OUTPUT}")
       set(_msg "${_msg} - Failed")
```

Comment:
Can this be removed now? Alternatively this should be changed to use `hpx_debug()`.